Libplanet/Blockchain/BlockChain.cs

```diff
@@ -548,7 +548,8 @@ bool renderActions
                 HashDigest<SHA256>? tip = Store.IndexBlockHash(Id, -1);
 
                 var nonceDeltas = new Dictionary<Address, long>();
-                foreach (Transaction<T> tx1 in block.Transactions)
+                var orderedTransactions = block.Transactions.OrderBy(tx => tx.Nonce);
```

Comment:
There is a bit of concern about preload performance. can we check if sorting on each addition affects it?

Reply:
Yes, this changes can cause to affect it.
But, IMHO I think the changes are a trade with memory.